packages/cli/src/commands/deploy/aws.js

```diff
@@ -68,28 +82,34 @@ export const handler = async ({ provider }) => {
         title: 'Building and Packaging...',
         task: () =>
           new Listr(
-            providerData.buildCommands.map((commandDetail) => {
-              return {
-                title: commandDetail.title,
+            [
+              {
+                title: providerData.buildCommands[0].title,
                 task: async () => {
-                  await execa(...commandDetail.command, {
+                  await execa(...providerData.buildCommands[0].command, {
                     cwd: BASE_DIR,
                   })
                 },
-              }
-            }),
+              },
+              {
+                title: 'packing',
+                task: pack,
+              },
+            ],
             { collapse: false }
           ),
       },
     ].filter(Boolean),
-    { collapse: false }
+    { collapse: false, renderer: verbose && VerboseRenderer }
   )
 
   try {
     await tasks.run()
 
     console.log(c.green(providerData.deployCommand.title))
-    const deploy = execa(...providerData.deployCommand.command, {
+    const deployCommand = [...providerData.deployCommand.command]
```

Comment:
This stuff doesn't look so good to me still...

Reply:
We really should just have all of the steps inside listr, and skip things if they're not required